.python-version

```diff
@@ -0,0 +1,4 @@
+2.7.15
```

Comment:
Fine for now, this might be dropped in the future since anyway python 2 will retire in 5 ~months.
Also for the part on statistics tests that I want to implement depends on `zfit` and is only compatible with python >= 3.5.

Reply:
Why pin down a particular version? And what is this file used for? Sorry if obvious ...